Project.toml

```diff
@@ -19,12 +19,10 @@ StaticArrays = "90137ffa-7385-5640-81b9-e52037218182"
 
 [compat]
 BlockArrays = "0.14"
-ContinuumArrays = "0.4"
```

Comment:
Why are these deleted?

Reply:
I was running into a dependency clash between branches that should now be resolved with your recent tags. I'll re-add the entry in the next commit.